doc/examples/ex52/ex52.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# GMT EXAMPLE 52
+#
+# Purpose:     Mixing images and overlaying on the Earth
+# GMT modules: grdgradient, grdimage, grdmath, grdmix, grdsample, solar
+
+gmt begin ex52 ps
```

Comment:
```suggestion
gmt begin ex52
```

Reply:
Sorry, it snuck back in to make the PS orig.